docs/includes/file-tutorial-excel-custom-functions.md

```diff
@@ -31,7 +31,7 @@ In this tutorial, you will:
 
 ## Create a custom functions project
 
-You’ll begin this tutorial by using the Yo Office generator to create the files that you need for your custom functions project.
+ You’ll begin this tutorial by using the Yo Office generator to create the files that you need for your custom functions project. If you have previously installed yo office, make sure to update your package to pull the latest from npm. You can do this by running `npm install -g yo generator-office`.
```

Comment:
@davidchesnut do you think this is sufficient to describe the process? We're trying to troubleshoot for people who go through this tutorial but already installed yo office and didn't realize they have to update.